pom.xml

```diff
@@ -46,7 +46,7 @@
 			${project.basedir}/src/main/resources/swagger.api/transformation-fields.yaml
 		</transformation-fields.yaml.file>
 
-		<folio-spring-base.version>3.0.3</folio-spring-base.version>
+		<folio-spring-base.version>5.0.2</folio-spring-base.version>
 		<lombok.version>1.18.18</lombok.version>
```

Comment:
Folio spring base 5.0.2 has 2.7.3 version  of spring-boot-starter-parent, so it better use here the same at pom.xml